Override ckConfigurator for local needs

jsconan opened the pull request:
depends on : https://github.com/oat-sa/tao-core/pull/329

Override ckConfigurator for local needs:
- qtiCommonRendered : allow plugin qtiUnderline
- qtiCreator : inject MathJax dependency, allow plugins qtiImage, qtiInclude, qtiUnderline, MathJax

krampstudio commented on views/js/qtiCreator/helper/ckConfigurator.js:
```diff
@@ -0,0 +1,54 @@
+/**
+ * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
+ */
+define(['ui/ckeditor/ckConfigurator', 'mathJax'], function(ckConfigurator, mathJax) {
```
missing `use strict;`

krampstudio commented:
Check also for missing `use strict` and wrong header everywhere

jsconan commented:
I added missing 'use strict'; and fixed wrong headers

2015-05-21 11:45 GMT+02:00 Bertrand CHEVRIER <notifications@github.com>:

> Check also for missing use strict and wrong header everywhere
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oat-sa/extension-tao-itemqti/pull/264#issuecomment-104205234>
> .
>



-- 




*Jean-Sébastien CONAN*
Software Engineer

•   Email:   jean-sebastien.conan@vesperiagroup.com
•   GSM: (+352) 621 320 343

*Vesperia Group*
148, Route d'Arlon
L-8010 Strassen
Luxembourg

http://www.vesperiagroup.com/
